Cartography/Constrain.swift

```diff
@@ -8,139 +8,188 @@
 
 import Foundation
 
-/// Updates the constraints of a single view.
+/// Removes all constraints for a group.
 ///
-/// - parameter view:    The view to layout.
-/// - parameter replace: The `ConstraintGroup` whose constraints should be
-///                      replaced.
-/// - parameter block:   A block that declares the layout for `view`.
+/// - parameter clear: The `ConstraintGroup` whose constraints should be removed.
 ///
-@discardableResult public func constrain(_ view: View, replace group: ConstraintGroup? = nil, block: (LayoutProxy) -> ()) -> ConstraintGroup {
-    let constraintGroup = group ?? ConstraintGroup()
-    let context = Context()
-    block(LayoutProxy(context, view))
-    constraintGroup.replaceConstraints(context.constraints)
+public func constrain(clear group: ConstraintGroup) {
+    group.replaceConstraints([])
+}
 
-    return constraintGroup
+@discardableResult public func constrain<A: LayoutElement>(elements element: A, replace group: ConstraintGroup = .init(), block: (A.ProxyType) -> Void) -> ConstraintGroup {
```

Comment:
For all the `constraint` functions you have named the first parameter as `elements` this will cause all the current user to change all the source code in order to compile against the new version.

Reply:
And also for the case where we have just a single view being constrained this named doesn't make sense for the parameter.

Reply:
This has been changed 👍